Source/AppAuth/macOS/OIDExternalUserAgentMac.h

```diff
@@ -29,6 +30,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface OIDExternalUserAgentMac : NSObject <OIDExternalUserAgent>
 
+/*! @brief The designated initializer.
+    @param presentingWindow The window from which to present the ASWebAuthenticationSession.
+ */
+- (instancetype)initWithPresentingWindow:(NSWindow *)presentingWindow API_AVAILABLE(macosx(10.15));
```

Comment:
We'd want to make this available for all macOS versions, mark it as the designated initializer, and deprecate -init.  See the iOS implementation for an example.

Reply:
Done.